Reference:	https://github.com/pypa/pip-audit/pull/715

--- pip_audit/_format/cyclonedx.py.orig	2023-12-19 03:49:32 UTC
+++ pip_audit/_format/cyclonedx.py
@@ -90,7 +90,7 @@ class CycloneDxFormat(VulnerabilityFormat):
             logger.warning("--fix output is unsupported by CycloneDX formats")
 
         bom = _pip_audit_result_to_bom(result)
-        formatter = output.get_instance(
+        formatter = output.make_outputter(
             bom=bom,
             output_format=self._inner_format.value,
             schema_version=output.SchemaVersion.V1_4,
--- pyproject.toml.orig	2023-12-19 03:49:32 UTC
+++ pyproject.toml
@@ -28,7 +28,7 @@ classifiers = [
 ]
 dependencies = [
     "CacheControl[filecache] >= 0.13.0",
-    "cyclonedx-python-lib >= 4,< 6",
+    "cyclonedx-python-lib >= 5,< 7",
     "html5lib>=1.1",
     "packaging>=23.0.0",                 # https://github.com/pypa/pip-audit/issues/464
     "pip-api>=0.0.28",
